cirq-core/cirq/ops/dense_pauli_string.py

```diff
@@ -614,12 +614,11 @@ def _as_pauli_mask(val: Iterable[cirq.PAULI_GATE_LIKE] | np.ndarray) -> np.ndarr
 
 
 def _attempt_value_to_pauli_index(v: cirq.Operation) -> tuple[int, int] | None:
-    if not isinstance(v, raw_types.Operation):
+    if isinstance(v, pauli_string.PauliString):
+        pass
+    elif (v := pauli_string._try_interpret_as_pauli_string(v)) is None:
         return None
 
-    if not isinstance(v.gate, pauli_gates.Pauli):
```

Comment:
Remove this check since PauliString[q] is always `Pauli`

Reply:
Actually can you replace it with a check `len(v qubits) == 1`?

Reply:
Done